.github/sync-files.yaml

```diff
@@ -1,7 +1,17 @@
 - repository: autowarefoundation/autoware
   files:
+    - source: CODE_OF_CONDUCT.md
+    - source: CONTRIBUTING.md
+    - source: DISCLAIMER.md
+    - source: LICENSE
+    - source: .github/ISSUE_TEMPLATE/bug.yaml
+    - source: .github/ISSUE_TEMPLATE/config.yml
+    - source: .github/ISSUE_TEMPLATE/task.yaml
+    - source: .github/PULL_REQUEST_TEMPLATE.md
+    - source: .github/PULL_REQUEST_TEMPLATE/small-change.md
+    - source: .github/PULL_REQUEST_TEMPLATE/standard-change.md
     - source: .github/dependabot.yaml
-    - source: .github/workflows/backport.yaml
```

Comment:
Need to check .github/workflows/backport.yaml

Reply:
Need backport
https://github.com/tier4/autoware_launch-old/pull/477

Reply:
Add it by https://github.com/tier4/autoware_launch/pull/28/commits/a6d8f1bd050cce4525eff49d661b00ed7773cd7a